src/config/exocom_production_dependency.go

```diff
@@ -47,6 +48,13 @@ func (e *exocomProductionDependency) GetDeploymentServiceEnvVariables(secrets ty
 // GetDeploymentVariables returns a map from string to string of variables that a dependency Terraform module needs
 func (e *exocomProductionDependency) GetDeploymentVariables() (map[string]string, error) {
 	exocomDevelopmentDependency := &exocomDevelopmentDependency{types.DevelopmentDependencyConfig{}, e.appConfig, e.appDir}
-	serviceRoutes, err := exocomDevelopmentDependency.getServiceRoutesString()
-	return map[string]string{"exocom_service_routes": serviceRoutes}, err
+	serviceRoutes, err := exocomDevelopmentDependency.compileServiceRoutes()
+	if err != nil {
+		return map[string]string{}, err
+	}
+	serviceRoutesJSON, err := json.Marshal(serviceRoutes)
+	if err != nil {
+		return map[string]string{}, err
+	}
+	return map[string]string{"SERVICE_ROUTES": string(serviceRoutesJSON)}, err
```

Comment:
Can we please add a comment (or even better a test) so we know why this encoding in this way?